pykafka/rdkafka/producer.py

```diff
@@ -0,0 +1,163 @@
+from concurrent import futures
```

Comment:
We should pull out the producer futures logic prior to merge, since that interface is still being debated and experimented with.